Ms graph user permissions bug fix (#18053)
demisto/content: Packs/MicrosoftGraphUser/ReleaseNotes/1_4_2.md

#### Integrations
##### Azure Active Directory Users

> - Added *client credentials flow* as another authentication flow for a self-deployed configuration.

Reviewer: ```suggestion
- Added the Client Credentials authentication flow for self deployed Azure App configuration.
```

> - Fixed an issue where the ***msgraph-user-session-revoke*** command did not work properly with the *authorization code flow*.

Reviewer: i think we can remove the second bullet
